spec/rake_tasks/abother_dummy_rake_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rake'
+
+describe 'Another dummy rake' do
+  describe "another_dummy:do_something_once" do
+    let(:task_name) { "another_dummy:do_something_once" }
+    let(:task) { Rake::Task[task_name] }
+
+    context 'when Rake.load_rakefile is used to load rake task' do
+      before(:all) do # :all must be used to ensure the rake task is loaded only once
+        Rake.load_rakefile("tasks/another_dummy.rake")
```

Comment:
This must be a rake task with a different name to avoid conflict with the `tasks/another_dummy.rake` rake task.